wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/exception/NotAGitRepositoryException.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.git.exception;
+
+import java.util.Map;
+import org.eclipse.che.api.core.rest.shared.dto.ServiceError;
+
+public class NotAGitRepositoryException extends GitException {
```

Comment:
I guess it is better GitInvalidRepsitoryException

Reply:
done